.travis.yml

```diff
@@ -18,8 +18,7 @@ addons:
 
 env:
   global:
-  - DJANGO_SETTINGS_MODULE=openzaak.conf.test
-  - SECRET_KEY=dummy
+  - DJANGO_SETTINGS_MODULE=openzaak.conf.envs.ci
```

Comment:
Can you please reverse this? Add config "parts" in a package "includes" and keep the config targets in `conf`.

Reply:
Yep :+1: